addons/website_event/views/event_svg.xml

```diff
@@ -0,0 +1,56 @@
+            <path d="M58.67 115.67C58.11 115.23 57.31 115.32 56.87 115.88C56.43 116.43 56.52 117.24 57.07 117.68L61.27 121.02C61.51 121.21 61.79 121.3 62.07 121.3C62.45 121.3 62.82 121.13 63.07 120.82C63.51 120.27 63.42 119.46 62.87 119.02L58.67 115.68V115.67Z" />
+            <path d="M50.27 108.99C49.72 108.55 48.91 108.64 48.47 109.19C48.03 109.74 48.12 110.55 48.67 110.99L52.87 114.33C53.11 114.52 53.39 114.61 53.67 114.61C54.04 114.61 54.42 114.44 54.67 114.13C55.11 113.58 55.02 112.77 54.46 112.33L50.26 108.99H50.27Z" />
+            <path d="M67.07 122.34C66.52 121.9 65.71 121.99 65.27 122.55C64.83 123.1 64.92 123.91 65.47 124.35L69.67 127.69C69.91 127.88 70.19 127.97 70.47 127.97C70.84 127.97 71.22 127.8 71.47 127.49C71.91 126.94 71.82 126.13 71.26 125.69L67.06 122.35L67.07 122.34Z" />
+            <path d="M75.47 129.02C74.92 128.58 74.11 128.67 73.67 129.22C73.23 129.77 73.32 130.58 73.88 131.02L78.08 134.36C78.32 134.55 78.6 134.64 78.88 134.64C79.25 134.64 79.63 134.47 79.88 134.16C80.32 133.61 80.23 132.8 79.68 132.36L75.48 129.02H75.47Z" />
+            <path d="M83.87 135.7C83.31 135.26 82.51 135.35 82.07 135.91C81.63 136.47 81.72 137.27 82.28 137.71L86.48 141.05C86.72 141.24 87 141.33 87.28 141.33C87.66 141.33 88.03 141.16 88.28 140.85C88.72 140.3 88.63 139.49 88.07 139.05L83.87 135.71V135.7Z" />
+            <path d="M93.47 146.26C93.19 146.26 92.91 146.17 92.67 145.98L90.66 144.38C90.1 143.94 90.02 143.13 90.46 142.58C90.9 142.03 91.71 141.94 92.26 142.38L94.27 143.98C94.83 144.42 94.91 145.23 94.48 145.78C94.23 146.1 93.85 146.26 93.48 146.26H93.47Z" />
+            </g>
+            </g>
+            <defs>
+            <clipPath id="clip0_56_106">
+            <rect width="210.97" height="179" fill="white"/>
```

Comment:
Question: will this white rect cover any page's background color?
If yes we should avoid that I guess.

Reply:
Removed it, didn't saw any changes with or without 🤔